### ISSUE TITLE
Bug in FW or wrong flight controller?

### ISSUE THREAD

**Issue by romko1** [iNavFlight/inav#4427]
Hello,
Right after uploading FW 2.1.0 for MATEKF722mini and when I plug in an inav configurator I2C error 45 reports.
In CLI console - resource: I see flawed rows:
C08: G÷?ÊRúáL¾ ø|ïhÌ®¡Ø¿?¯?¿»óS¸eãHïÒÜm¼´Üæbz0àå?n¥ö¥n¢1?Ç?i`4?»Ö%þ?Ãñ?_?÷uÖ?õ?'H ¶oÞng?Q`?üTßCìä?MºrÝqöú6òôOjSiriO^,·o©+??UKLg/.+ªÿ¾Ó?òs¡mDjyónÍ²Ö½k	C1Ù;ãÿì«"V0ýÅW5ÆEÇmS?Ý±}cØæËÆ Zx ÏÞé×1 OUT
C09: G÷?ÊRúáL¾ ø|ïhÌ®¡Ø¿?¯?¿»óS¸eãHïÒÜm¼´Üæbz0àå?n¥ö¥n¢1?Ç?i`4?»Ö%þ?Ãñ?_?÷uÖ?õ?'H ¶oÞng?Q`?üTßCìä?MºrÝqöú6òôOjSiriO^,·o©+??UKLg/.+ªÿ¾Ó?òs¡mDjyónÍ²Ö½k	C1Ù;ãÿì«"V0ýÅW5ÆEÇmS?Ý±}cØæËÆ Zx ÏÞé×2 OUT

Is this a FW error or is the board - flight controller itself wrong?
Thank you.
Roman

**Comment by shellixyz** [comment]
I can't reproduce the issue. It is working fine for me configurator version 2.1.4 and FW 2.1.0 final. Make sure you flashed with "full chip erase" selected and you're using the right baud rate. The default is 115200.

**Comment by romko1** [comment]
Thank you, I'll take the recommended steps.

**Comment by romko1** [comment]
Unfortunately, FW inav also keeps telling me that the F7 MATEKF22mini board has I2C 45 errors.... I also noticed that the memory is only 512kB....

**Comment by romko1** [comment]
2019-02-28 @ 20:33:13 -- Detected device with total flash size 512 kiB
2019-02-28 @ 20:33:13 -- Read protection not active
2019-02-28 @ 20:33:21 -- Erased 512 kB of flash successfully
2019-02-28 @ 20:33:41 -- USB device successfully closed

**Comment by b14ckyy** [comment]
> Unfortunately, FW inav also keeps telling me that the F7 MATEKF22mini board has I2C 45 errors.... I also noticed that the memory is only 512kB....

That's correct. That is the firmware flash size. Not the Blackbox flash. 

The output errors come from a wrong baud rate setting. Get the same issue when connecting with the wrong baud rate, if i forget to switch in configurator after using Bluetooth on 57600

**Comment by romko1** [comment]
I've made the recommended modifications, but I'm unsuccessful. I attach a few print-screens.
![baud_rate](https://user-images.githubusercontent.com/34416051/53603289-4b4b7f80-3bb1-11e9-9603-e85342af47e6.jpg)
![ports_inav_210](https://user-images.githubusercontent.com/34416051/53603295-556d7e00-3bb1-11e9-92e9-00f50e5c7593.jpg)
![status_matekf722](https://user-images.githubusercontent.com/34416051/53603306-5c948c00-3bb1-11e9-98f3-d2c18edf450d.jpg)

**Comment by shellixyz** [comment]
Sorry I didn't get that you were talking about the `resource` command. Confirmed there is a software issue there.

**Comment by shellixyz** [comment]
Will be fixed in the next release. Thank you for reporting. C8 and C9 are the pinIO pins.

**Comment by romko1** [comment]
OK. But my question is: C08 and C09 have been corroborated, but related to the I2C errors 45?
In the CLI console: status:
![status_matekf722](https://user-images.githubusercontent.com/34416051/53622034-c84e1780-3bf8-11e9-8956-7fb37d70487a.jpg)

I did a little test. I uploaded my FC MATEKF722 mini -> FW Betaflight 3.5.6 and in the CLI console (betaflight it allows) eliminated the resource for the I2C1 B08 et B09 and the I2C error disappeared ... maybe it looks to the board error, or? Thank you.
